### ISSUE TITLE
Cultivar parameter issue

### ISSUE THREAD

**Issue by zheng-jinhui** [APSIMInitiative/ApsimX#9568]
### What happened?

Dear friend,

Recently, I encountered some issues while using APSIM Next Generation (APSIMNG), particularly when defining cultivar parameters. I found that some parameters mentioned in research papers are not available in the software.

For instance, I referred to the official APSIMNG technical documentation (https://apsimdev.apsim.info/ApsimX/Releases/2020.11.09.5842/Wheat.pdf),  which lists widely used cultivar parameters such as [Phenology].MinimumLeafNumber.FixedValue and [Phenology].VrnSensitivity.FixedValue. However, when I tried to locate these parameters in the software, they were nowhere to be found. This made me wonder whether these parameter names have been updated in the newer versions of the software or if the technical documentation hasn’t been synchronized with the latest release.

This issue has caused significant difficulties in defining the cultivar parameters for my work. I hope to receive your guidance and a solution to this problem.

Looking forward to your response!

Best regards,
Zheng

### What Operating System are you on?

Windows

**Comment by BrianCollinss** [comment]
How do you look for them?

**Comment by zheng-jinhui** [comment]
Dear Dr. [Brian Collins](https://github.com/BrianCollinss),

Thank you for your reply.

As shown in the image, I have set up a "Replacements" module where I placed the crop parameters I need to replace. However, under the [Phenology] module, I couldn't find the parameters [Phenology].VrnSensitivity.FixedValue as described in the technical documentation (https://apsimdev.apsim.info/ApsimX/Releases/2020.11.09.5842/Wheat.pdf).  

Additionally, I am currently using the October 2024 version, but after updating to the latest December 2024 version, I noticed that the previously available parameter [Phenology].EarlyFlowering.Target.FixedValue is no longer present. Is this behavior expected? Or has the updated model replaced these parameters with more advanced modules? If that is the case, could you kindly provide a new technical document to explain the changes?  

Thank you for your assistance! 

Looking forward to your response!

Best regards,
Jinhui Zheng

![Snipaste_2024-12-30_15-26-02](https://github.com/user-attachments/assets/f19c95e4-808e-43b7-8abd-449f79c9c347)
![image](https://github.com/user-attachments/assets/807a8ad2-4315-4fbc-a5b0-8d8399bd516a)

**Comment by BrianCollinss** [comment]
I recommend you right click on the Wheat model that is placed under Simulation and select Show Model Structure. Then you can investigate the model. Under phrenology you will be find the stages currently available in the wheat model.

**Comment by zheng-jinhui** [comment]
Dear Dr. [Brian Collins](https://github.com/BrianCollinss),

Thank you for providing the suggested solutions.  

However, as shown in the image, I still cannot find the parameters I mentioned earlier. Additionally, I noticed that some parameters lack proper descriptions, and I couldn’t locate any relevant information about certain parameters, such as *PhyllochronPpSensitivity*, in the official technical documentation. This has caused significant difficulties in the process of writing my research paper.  

I would greatly appreciate your further guidance on this matter. Thank you in advance!

Best regards,
Jinhui Zheng

![image](https://github.com/user-attachments/assets/0e3c0ad3-4825-4b47-b3af-cc8ef71f71c4)

**Comment by BrianCollinss** [comment]
I assume that parameter belongs to an older version of the wheat model. If you click on the "Memo"s under phrenology, you will find some information about the current vernalization/photo period model implemented in Wheat along with the title of the relevant paper(s).

**Comment by zheng-jinhui** [comment]
Yes, I suspect that the model version update might have replaced these parameters. I understand that most parameters have a "Memo" section to store detailed information, but for a few, such as *PhyllochronPpSensitivity*, I indeed couldn't find any related description or Memo.  

Thank you very much for your explanation. I hope the research team can release an updated technical document soon, as it would be of great help to us users.
